### PR TITLE
routes updated and clean

### PR DESCRIPTION
**Pull request by Pagehey**
Hey, checkout these awesome routes! =D
Special thx to Cecile!

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,17 +3,22 @@
 
   devise_for :users
 
-  resources :spaceships do # , only: [:index, :show, :create, :update, :destroy] do
-    # resources :reservations, only: [:new] unecessary as implemented on spaceships#show
+  # AS A RENTER
```
Comment (thbrillet) [inline config/routes.rb]: love this comment like wow

File: config/routes.rb [hunk 2]
```diff
@@ -3,17 +3,22 @@
...
+  resources :spaceships, only: [:index, :show] do
+    resources :reservations, only: [:create]
   end
 
-  post 'reservations/:id/accept', to: 'reservations#accept', as: :accept_reservation
-  post 'reservations/:id/decline', to: 'reservations#decline', as: :decline_reservation
-
-  resources :reservations, only: [:index, :create, :accept, :decline]
+  resources :reservations, only: [:index]
```
Comment (thbrillet) [inline config/routes.rb]: omg ggwp

File: config/routes.rb [hunk 3]
```diff
@@ -3,17 +3,22 @@
...
 
+  # AS A OWNER
   namespace :owner do
-    resources :reservations, only: [:index]
-    # resources :spaceships, only: [:new, :edit]
+    resources :spaceships, only: [:index, :new, :create]
+
+    resources :reservations, only: [:index] do
+      member do
```
Comment (thbrillet) [inline config/routes.rb]: nice namespacing right here boyz

File: config/routes.rb (hunk 1 above)
Comment (JulieYende) [reply config/routes.rb]: All good 👍

File: config/routes.rb (hunk 3 above)
Comment (JulieYende) [reply config/routes.rb]: Dope

**Comment by heyKV** [comment]
oki pour moi!

**Pull request merged**